### ISSUE TITLE
El::MaxAbs(const AbstractDistMatrix<F> &A) not implemented, but method definition is

### ISSUE THREAD

**Issue by davidsblom** [elemental/Elemental#168]
When I call the routine:
``` c++
double maxAbs = El::MaxAbs( matrix );
```
the compilation is successfull, but the linker cannot find the implementation.

```
libRBFMeshMotionSolver.so: undefined reference to `El::BaseHelper<double>::type El::MaxAbs<double>(El::AbstractDistMatrix<double> const&)'
collect2: error: ld returned 1 exit status
```

I digged around in the code, and found that it is still a todo. 

I've implemented my own version like this by reusing some code of the Max() method.

``` c++
template<typename Real>
Real MaxAbs( const El::DistMatrix<Real> & A )
{
    Real value = 0;

    if ( A.Participating() )
    {
        // Store the index/value of the local pivot candidate
        const El::Int mLocal = A.LocalHeight();
        const El::Int nLocal = A.LocalWidth();
        const Real * ABuf = A.LockedBuffer();
        const El::Int ALDim = A.LDim();

        for ( El::Int jLoc = 0; jLoc < nLocal; ++jLoc )
            for ( El::Int iLoc = 0; iLoc < mLocal; ++iLoc )
                value = El::Max( std::abs( value ), ABuf[iLoc + jLoc * ALDim] );

        value = El::mpi::AllReduce( value, El::mpi::MAX, A.DistComm() );
    }

    El::mpi::Broadcast( value, A.Root(), A.CrossComm() );
    return value;
}
```

**Comment by rhl-** [comment]
Issue a PR and we can review it :)

**Comment by davidsblom** [comment]
Fixed in #169. Thanks!
